org.uqbar.project.wollok/src/org/uqbar/project/wollok/messages.properties

```diff
@@ -37,6 +37,7 @@ WollokScopeProvider_unresolvedImport = Error while resolving import {0}
 WollokDslValidator_BAD_USAGE_OF_IF_AS_BOOLEAN_EXPRESSION = Bad usage of if ! You must return the condition itself without using if. Use: 
 WollokDslValidator_IF_USED_IN_AN_EXPRESSION_SHOULD_HAVE_AN_ELSE_STATEMENT = If-statements without else cannot be considered valid expressions as they cannot always yield a value
 WollokDslValidator_EXPECTING_BOOLEAN = Expecting a boolean
+WollokDslValidator_EXPECTING_BOOLEAN_COMPARING_VS_ASSIGNING = You did an assignment inside an if condition. Maybe you wanted to use '==' instead of '=' ? 
```

Comment:
```suggestion
WollokDslValidator_EXPECTING_BOOLEAN_COMPARING_VS_ASSIGNING = You did an assignment inside an if condition. This won't work, a boolean expression is expected. Maybe you wanted to use '==' instead of '=' ? 
```